services/istio/1.9.1/helmrelease.yaml

```diff
@@ -16,6 +16,5 @@ spec:
     name: management
     namespace: kommander-flux
   postBuild:
-    substituteFrom:
-      - kind: ConfigMap
-        name: substitution-vars
+    substitute:
+      releaseNamespace: ${releaseNamespace}
```

Comment:
are any of the other vars in the cm used? we can still include `substituteFrom` with this change, the `substitute` values would take precedence over the cm

> The var values which are specified in-line with substitute take precedence over the ones in substituteFrom.

https://fluxcd.io/docs/components/kustomize/kustomization/#variable-substitution

Reply:
Thanks Grace! I couldn't find this info (about priorities). I will merge it as it is since it worked yesterday 🙂